bool querry
float64 position
---
float64 servoPos